certified-connectors/CognitiveServicesSpeechToText/apiDefinition.swagger.json

```diff
@@ -0,0 +1,1952 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "CognitiveServicesSpeechToText",
+    "description": "Azure Cognitive Services for Batch Speech-to-text",
+    "version": "1.0",
+    "contact": {
+      "name": "Cognitive Services - Speech Services",
+      "url": "https://docs.microsoft.com/azure/cognitive-services/speech-service/support"
+    }
+  },
+  "host": "eastus.api.cognitive.microsoft.com",
```

Comment:
@RichardSunMS the host URL should be in production. can you please confirm if that is the case here?

Reply:
@Amjed-Ayoub , yes I confirm this host is already in production. But one quick question here: we will have different regional hosts here, how to make this as a parameter in future?

Reply:
@RichardSunMS thank you so much for the confirmation. 
Regarding your question please see how to [Set Host URL](https://learn.microsoft.com/en-us/connectors/custom-connectors/policy-templates/dynamichosturl/dynamichosturl)